styles/globals.css

```diff
@@ -137,7 +137,7 @@ body[data-theme='dark'] {
   --color-green: #2cb67d;
 }
 
-body[data-theme='light'] {
+body.light-mode {
```

Comment:
The library adds this change as it checks scheme preferences depending on the system